src/models/site/usePutSiteSettings.js

```diff
@@ -22,7 +24,7 @@ export default function usePutSiteSettings() {
       const successful = get(response, ['data', 'success'], false);
       setLoading(false);
       if (successful) {
-        dispatch(setSiteSettingsNeedsFetch(true));
+        queryClient.invalidateQueries(queryKeys.settingsConfig);
```

Comment:
So, after invalidation, how do these query keys become valid again?

Reply:
"invalidating" the settings config is basically saying "the settings config is now stale because we just updated it"

telling react-query this will cause it to automatically begin refetching the settings config